lib/rubycritic/core/analysed_module.rb

```diff
@@ -27,6 +28,17 @@ def rating
       @rating ||= Rating.from_cost(cost)
     end
 
+    def lines
+      return @lines if @lines
+      output = `wc -l < "#{path}"`.to_i
+      result = $?.success?
+      if result
+        @lines ||= output
+      else
+        @lines ||= File.open(path).lines.count
+      end
+    end
```

Comment:
`lines` was deprecated in favor of `each|each_line`.

I did a little [benchmark on Counting number of lines of a file in Ruby](https://gist.github.com/guilhermesimoes/d69e547884e556c3dc95) and `File.read(path).each_line.count` won.

However, `wc` was so slow that I'm thinking there's something wrong with it. Could you please run the benchmarks as well? If `wc` is not the fastest, I think we could simplify this to just using Ruby.